06_puppy/mohankrishna/storer_test.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type storerImpl int
+
+const (
+	maps  storerImpl = iota
+	smaps storerImpl = iota
```

Comment:
while this works, it is more idiomatic go to just say `smaps` here, so the whole const block reads:
```
const (
    maps storerImpl = iota
    smaps
)
```

See https://github.com/golang/go/wiki/Iota for an example and links to specs, etc.